tests/org.palladiosimulator.dataflow.confidentiality.analysis.tests/src/org/palladiosimulator/dataflow/confidentiality/analysis/AnalysisFeatureTest.java

```diff
@@ -0,0 +1,56 @@
+package org.palladiosimulator.dataflow.confidentiality.analysis;
+
+import static org.palladiosimulator.dataflow.confidentiality.analysis.AnalysisUtils.TEST_MODEL_PROJECT_NAME;
+
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.palladiosimulator.dataflow.confidentiality.analysis.testmodels.Activator;
+
+@TestInstance(Lifecycle.PER_CLASS)
+public class AnalysisFeatureTest {
```

Comment:
Just a reminder to not forget to merge the main branch with your updated code into this